rosidl_typesupport_c/include/rosidl_typesupport_c/message_type_support_dispatch.h

```diff
@@ -24,6 +24,17 @@ extern "C"
 {
 #endif
 
+/// Get the message type support handle specific to this identifier
```

Comment:
Nitpick: the first sentence of the docblock should end with a period.

Reply:
Fixed here and below